internal/keycloakb/validation.go

```diff
@@ -64,3 +66,18 @@ func ValidateParameterDate(prmName string, value *string, dateLayout string, man
 	}
 	return nil
 }
+
+// ValidateParameterTimestamp validates a datetime integer
+func ValidateParameterTimestamp(prmName string, value *int64, mandatory bool) error {
```

Comment:
A timestamp can be valid either in the past or in the future. You should be more precise with the name of this method so that we can understand if you are checking a date that must be in the past (birthdate) or must be in the future (expiry time)

Reply:
This method doesn't exist anymore